uber/models.py

```diff
@@ -749,9 +749,16 @@ def model_mixin(cls, model):
             else:
                 raise ValueError('No existing model with name {}'.format(model.__name__))
 
-        for attr in dir(model):
-            if not attr.startswith('_'):
-                setattr(target, attr, getattr(model, attr))
+        for name in dir(model):
+            if not name.startswith('_'):
+                attr = getattr(model, name)
+                if name in target.__table__.c:
```

Comment:
When we say something like

```python
@Session.model_mixin
class Attendee:
    foo = Column(Integer, default=0)
```

Then behind the scenes we place ``foo`` onto the ``Attendee`` class.  When this happens, SQLAlchemy automatically fills in a couple of fields on the ``Column`` object such as the ``table`` and the ``name``.  Here we need to fill those in manually before calling ``.replace()`` on the new column.